bin/llvm-kompile-clang

```diff
@@ -207,7 +219,7 @@ if [[ "$OSTYPE" == "darwin"* ]]; then
   start_whole_archive="-force_load"
   end_whole_archive=""
 
-  flags+=("-Wl,-flat_namespace" "-Wl,-undefined" "-Wl,dynamic_lookup")
```

Comment:
Globally-enabled `-flat_namespace` is removed here @Robertorosmaninho

Reply:
Perfect! Thank you for the detailed explanation, Bruce!